network/requestTracker.go

```diff
@@ -518,13 +521,29 @@ func (rt *RequestTracker) getForwardedConnectionAddress(header http.Header) (ip
 	if rt.config.UseXForwardedForAddressField == "" {
 		return
 	}
-	forwardedForString := header.Get(rt.config.UseXForwardedForAddressField)
+	var forwardedForString string
+	// if we're using the standard X-Forwarded-For header(s), we need to parse it.
+	// as UseXForwardedForAddressField defines, use the last value from the last X-Forwarded-For header's list of values.
```

Comment:
Why wouldn't we apply the same rules (use the last header, and the last value in that header) when the header has another name?

Reply:
I guess we can

Reply:
This stands out as a rule for X-Forwarded-For, but not sure it's consistent across all headers general case.